addons/mail/models/mail_scheduled_message.py

```diff
@@ -332,7 +332,7 @@ def _post_messages_cron(self, limit=50):
     def _to_store_defaults(self):
         return [
             Store.Many("attachment_ids"),
-            Store.One("author_id", rename="author"),
+            Store.One("author_id", ["name", "user_id", "avatar_128"]),
```

Comment:
`user_id` is never the field we want in partner.
Current store asks for `user` instead (later it should be `user_ids` or `main_user_id` if we finally introduce the field).

Reply:
I don't know maybe we should just drop this suggestion and keep the default. It's not like we have many default fields in partner store.